server/routes/handlers/outcomes/getCasesInProgressHandler.js

```diff
@@ -1,12 +1,13 @@
+const getOutcomeTypesListFilters = require('../../../utils/getOutcomeTypesListFilters')
 
 const getCasesInProgressHandler = caseService => async (req, res) => {
   const {
     params: { courtCode, title, sorts, state },
     params
   } = req
 
-  const filters = []
-  const filtersApplied = 0
+  const filters = getOutcomeTypesListFilters(req.query)
```

Comment:
Can these 2 lines as well go into the middleware?

Reply:
In fact I can see everything except render part can go into the middleware?

Did we ever need just one handler mapped to all 3 routes? (not precious about this)

Reply:
Technically we can now if we so wish? Again I'm not precious about it either, I just felt having it all in one was breaking the single responsibility principle (SRP)? Also I did the filters this way deliberately so each controller can control which filters get rendered (as cases to result tab has limited filters in comparison to the in-progress tab)